packages/@lwc/rollup-plugin/src/__tests__/resolver/resolver.spec.ts

```diff
@@ -37,8 +41,18 @@ async function runRollup(
 
 describe('resolver', () => {
     it('should be capable to resolve all the base LWC module imports', async () => {
-        const { warnings } = await runRollup('lwc-modules/lwc-modules.js');
+        const { warnings } = await runRollup('lwc-modules/lwc-modules.js', { external: [] });
+        expect(warnings).toHaveLength(0);
+    });
```

Comment:
This test wasn't doing its job because the base modules were configured as external, which means rollup wouldn't emit warnings that would've pointed to the regression.